core/schema/src/main/liquibase/26.0.0/changelog.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" >
+<changeSet author="patrick-schweizer" id="26.0.0-add-timescale-columns">
+    <sql>
+        CREATE TABLE timeseries(
+        time       TIMESTAMPTZ      NOT NULL,
+        context    TEXT              NOT NULL,
+        resource   TEXT              NOT  NULL,
+        name       TEXT              NOT  NULL,
+        type       TEXT              NOT  NULL,
+        value      DOUBLE PRECISION  NULL);
+    </sql>
+    <sql><!-- turn table into a timescale table, timescale plugin must be enabled in database -->
+        SELECT create_hypertable('timeseries', 'time');
```

Comment:
We'll need to find another way to do this if we proceed with the Timescale implementation, otherwise it will become a hard requirement. Maybe we can add a shell command? :D

Reply:
done: https://github.com/OpenNMS/opennms/pull/2845/files#diff-a66c4cc8fcbf80e198c136e3c1425741